resources/partial/workflow-action-springboot/${id}-workflow-action.client-extension-config.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
So i guess we need to have a followup task for pushing this to gradle task and generalize that pattern.

Reply:
yes!